dashboard/app/helpers/plc/tasks_helper.rb

```diff
@@ -1,6 +1,6 @@
 module Plc::TasksHelper
   def options_for_plc_tasks
     #Probably better to hardcode this - this list isn't going to change that much
-    [['Learning Resource', 'Plc::LearningResourceTask'],['Script Completion Task', 'Plc::ScriptCompletionTask']]
+    [['Learning Resource', 'Plc::LearningResourceTask'],['Script Completion Task', 'Plc::ScriptCompletionTask'], ['Written Assignment Task', 'Plc::WrittenAssignmentTask']]
```

Comment:
You might be able to clean this up with `Plc::Task.subclasses`, but that depends on whether or not they've been loaded yet and it may be more trouble than it's worth.